config/config.exs

```diff
@@ -1,6 +1,6 @@
 # This file is responsible for configuring your application
 # and its dependencies with the aid of the Mix.Config module.
-use Mix.Config
+import Config
```

Comment:
If the issue is a compiler warning because `use Mix.Config` is deprecated, just delete this file in this PR. Nothing is actually configured and I'm sure this file only exists because of a mix generator.

Reply:
Config is now deleted.